### PR TITLE
feat: Support ESLint 7.x

### LINKED ISSUE #45: Support ESLint 7.x
ESLint v7.0.0 is [released](https://eslint.org/blog/2020/05/eslint-v7.0.0-released) 🎉

It would be awesome to have official ESLint 7 support. 👊
I'm happy to help where I can of course 🙂

### PR DESCRIPTION
**Pull request by MichaelDeBoey**
BREAKING CHANGE: Requires Node@^10.12.x || 12.x
BREAKING CHANGE: Requires ESLint@^7.x

Closes #45

### CODE REVIEW

**Comment by benmonro** [comment]
Lgtm once the build is passing. Let me know if Travis is being a pain I've switched a lot of my projects to GitHub actions and been pretty happy.  Wouldn't mind doing that here of this gets out of line

**Comment by MichaelDeBoey** [comment]
@benmonro I think `kcd-scripts` can benefit from GitHub Actions eventually, but we're good for now with Travis 🙂

Will take a closer look on what's causing CI to fail

File: .travis.yml [hunk 1]
```diff
@@ -3,7 +3,9 @@ cache: npm
 notifications:
   email: false
 node_js:
+  - 10.12
```
Comment (benmonro) [inline .travis.yml:6]: @MichaelDeBoey looks like it's only failing on node 10.12, not sure why...

File: .travis.yml [hunk 2]
```diff
@@ -3,7 +3,9 @@ cache: npm
...
   - 10
+  - 12.0
```
Comment (benmonro) [inline .travis.yml:8]: is this 12.0 needed since we have 12 below it?

Comment (MichaelDeBoey) [reply .travis.yml:8]: 12.0 is specifically 12.0, while 12 is the latest 12.x

**Pull request merged**